### PR TITLE
Retention of commitment data

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/UpdateRetentionData.sql [hunk 1]
```diff
@@ -93,7 +93,7 @@ BEGIN TRANSACTION
 						Set @UpdateColsList = ''
 
 					Set @FromClause  = ' FROM ' + @DataSetSchema + '.' + @DataSetTable +' ST ' + ' JOIN ' + @RefDataSetSchema + '.' + @RefDataSetTable + ' PT '				
-
+					-- Set @JOINClause  = ' ON PT.' + @DatasetJOINColumn + ' = ST.' + @RefColumn				
 					Set @JOINClause  = ' ON PT.' + @RefColumn + ' = ST.' + @DataSetJOINColumn
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/UpdateRetentionData.sql:97]: why is this changed, will that not have an effect on previously configured tables.

Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/UpdateRetentionData.sql:97]: Ah Ignore, It's commented out.

**Review by uhimabindu — APPROVED** [review]

**Review by arvinthseran — APPROVED** [review]
